core-lightning/docker-compose.yml

```diff
@@ -57,9 +57,10 @@ services:
       - --bind-addr=${APP_CORE_LIGHTNING_DAEMON_IP}:9735
       - --addr=statictor:${TOR_PROXY_IP}:29051
       - --tor-service-password=${TOR_PASSWORD}
+      - --rpc-file=/data/.lightning/rpc/lightning-rpc
```

Comment:
Let remove this and use the default path which ends up on the host as `${UMBREL_ROOT}/app-data/core-lightning/data/lightningd/bitcoin/lightning-rpc`. That way we don't need to also update core-lightning and force users to update it before installing squeaknode-core-lightning to avoid issues.